script/utils/InvestableWorkstreams/01-DeployInvestableWorkstream.s.sol

```diff
@@ -0,0 +1,273 @@
+    bool hasDependency;
+    string[] dependencies = new string[](0);
+
+    // ========================================================================
+    // ENVIRONMENT VARIABLES OR CONSTANTS
+
+    uint orchestratorOwnerPrivateKey =
+        vm.envUint("ORCHESTRATOR_OWNER_PRIVATE_KEY");
+    address orchestratorOwner = vm.addr(orchestratorOwnerPrivateKey);
+
+    address collateralTokenAddress =
+        vm.envAddress("BONDING_CURVE_COLLATERAL_TOKEN");
+    ERC20 collateralToken = ERC20(collateralTokenAddress);
+
+    address bancorFormulaAddress = vm.envAddress("BANCOR_FORMULA_ADDRESS");
+    BancorFormula formula = BancorFormula(bancorFormulaAddress);
+
+    // ========================================================================
+    // BONDING CURVE PARAMETERS
+
+    bytes32 CURVE_TOKEN_NAME = "Conding Burve Token";
+    bytes32 CURVE_TOKEN_SYMBOL = "BCRG";
+    uint INITIAL_TOKEN_SUPPLY = 1;
+    uint INITIAL_COLLATERAL_SUPPLY = 1;
+    uint RESERVE_RATIO_FOR_BUYING = 330_000;
+    uint RESERVE_RATIO_FOR_SELLING = 330_000;
+    uint BUY_FEE = 0;
+    uint SELL_FEE = 100;
+    bool BUY_IS_OPEN = true;
+    bool SELL_IS_OPEN = false;
+
+    // ========================================================================
+
+    //-------------------------------------------------------------------------
+    // Storage
+
+    IOrchestrator _orchestrator;
+    address[] initialAuthorizedAddresses;
+
+    //-------------------------------------------------------------------------
+    // Script
+
+    function run() public override returns (address deployedOrchestrator) {
+        // ------------------------------------------------------------------------
+        // OPTIONAL PRE-STEPS
+
+        //If the factories aren't deployed on the target chain, we can run the deployment script to deploy the factories, implementations and Beacons.
+        address orchestratorFactory = DeploymentScript.run();
+
+        //If there's no formula deployment on the chain, we deploy the Bancor Formula
+        /* vm.startBroadcast(orchestratorOwnerPrivateKey);
+        {
+            formula = new BancorFormula();
+        }
+        vm.stopBroadcast();
+        */
+        // ------------------------------------------------------------------------
+        // Setup
+
+        // ------------------------------------------------------------------------
+        // Define Initial Configuration Data
+
+        // Orchestrator: Owner, funding token
+        IOrchestratorFactory.OrchestratorConfig memory orchestratorConfig =
+        IOrchestratorFactory.OrchestratorConfig({
+            owner: orchestratorOwner,
+            token: collateralToken
+        });
+
+        // Funding Manager: Virtual Supply Bonding Curve Funding Manager
+        IOrchestratorFactory.ModuleConfig memory
+            bondingCurveFundingManagerConfig = IOrchestratorFactory
+                .ModuleConfig(
+                bondingCurveFundingManagerMetadata,
+                abi.encode(
```

Comment:
The contract expect 2 objects and 1 token address, as well as the decimals of the issuance token. [See link](https://github.com/InverterNetwork/inverter-contracts/blob/dev/test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol#L163)